pkg/docker/containers-view.jsx

```diff
@@ -381,7 +382,13 @@ var ImageList = React.createClass({
     },
 
     render: function () {
-        var imageRows = this.state.images.map(function (image) {
+	var filtered = this.state.images.filter(function(image) {
+	    if (image.RepoTags[0].toLowerCase().indexOf(this.props.filterText.toLowerCase()) < 0)
```

Comment:
this would be a lot nicer if written directly:
`return (image.RepoTags[0].toLowerCase().indexOf(this.props.filterText.toLowerCase()) >= 0);`